dts/arm/st/h7/stm32h745Xi_m7.dtsi

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/h7/stm32h745.dtsi>
+
+/ {
+	cpus {
+		/delete-node/ cpu@1;
+	};
+
+	/* system data RAM accessible over over AXI bus */
+	sram0: memory@24000000 {
+		reg = <0x24000000 DT_SIZE_K(512)>;
+		compatible = "mmio-sram";
+	};
+
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/*
```

Comment:
why is that commented?

Reply:
This is the place holder for later ITCM introduction.

Reply:
done